### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by Centek96**
Made polish translation without specialized settings names.

### CODE REVIEW

File: src/qt_gui/translations/pl_PL.ts [hunk 1]
```diff
@@ -16,322 +16,322 @@
...
             <location filename="../about_dialog.ui" line="78"/>
             <source>shadPS4 is an experimental open-source emulator for the PlayStation 4.</source>
-            <translation>shadPS4 is an experimental open-source emulator for the PlayStation 4.</translation>
+            <translation>shadPS4 to eksperymentalny otwarto-źródłowy emulator PlayStation 4.</translation>
         </message>
         <message>
             <location filename="../about_dialog.ui" line="99"/>
             <source>This software should not be used to play games you have not legally obtained.</source>
-            <translation>This software should not be used to play games you have not legally obtained.</translation>
+            <translation>Nie powinien być używany w celu ogrywania nielegalnych kopii gier.</translation>
```
Comment (wojek07) [inline src/qt_gui/translations/pl_PL.ts:24]: Nit: Even thought entire sentence conveys appropriate message, I wonder whether used verb "ogrywać" is somewhat informal and as such not fits into vocabulary used in legal disclaimers?

File: src/qt_gui/translations/pl_PL.ts [hunk 2]
```diff
@@ -16,322 +16,322 @@
...
             <location filename="../game_install_dialog.cpp" line="74"/>
             <source>Error</source>
-            <translation>Error</translation>
+            <translation>Błąd</translation>
         </message>
         <message>
             <location filename="../game_install_dialog.cpp" line="75"/>
             <source>The value for location to install games is not valid.</source>
-            <translation>The value for location to install games is not valid.</translation>
+            <translation>Ścieżka jest nieprawidłowa.</translation>
```
Comment (wojek07) [inline src/qt_gui/translations/pl_PL.ts:78]: I think the source states something about the path and that information is omitted in translation.

File: src/qt_gui/translations/pl_PL.ts (hunk 1 above)
Comment (masermb) [reply src/qt_gui/translations/pl_PL.ts:24]: I'm backing it up as I had exactly the same feelings about it.

File: src/qt_gui/translations/pl_PL.ts (hunk 2 above)
Comment (masermb) [reply src/qt_gui/translations/pl_PL.ts:78]: I agree that there is a difference to the original, though the english source doesn't sound right and chosen translation might be the improvement overall.  Probably original should be  "Location of games directory is invalid." or  "Path to games location folder is invalid." etc.

**Comment by georgemoralis** [comment]
Already have #580 merged with this . Thanks for your efford ! If you have some instances with translation you can open a new pr

**Pull request closed without merging**